### PR TITLE
Towards #2522 - RandomizedPcaTrainer documentation

### PR DESCRIPTION
**Pull request by yaeldekel**
Adding documentation for Randomized PCA, as specified in #2252.

### CODE REVIEW

File: docs/api-reference/io-columns-anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Input and Output Columns
+The input features column data must be a known-sized vector of <xref:System.Single>. This trainer outputs the following columns:
+
+| Output Column Name | Column Type | Description|
+| -- | -- | -- |
+| `Score` | <xref:System.Single> | The score that was calculated by the anomaly detection model.|
```
Comment (shmoradims) [inline docs/api-reference/io-columns-anomaly-detection.md]: >  [](start = 38, length = 1)

if the range is (-inf,+inf) we call them 'unbounded score' to be clear that it's not normalized or anything like that.

File: src/Microsoft.ML.PCA/PcaTrainer.cs [hunk 2]
```diff
@@ -33,12 +33,35 @@ namespace Microsoft.ML.Trainers
...
+    /// | -- | -- |
+    /// | Machine learning task | Anomaly Detection |
+    /// | Is normalization required? | Yes |
+    /// | Is caching required? | No |
+    /// | Required NuGet in addition to Microsoft.ML | None |
+    ///
+    /// ### Training Algorithm Details
+    /// This trainer trains an approximate PCA using a randomized method for computing the singular value decomposition (SVD) of
+    /// the matrix whose rows are the input vectors.
+    /// The method is described [here](https://web.stanford.edu/group/mmds/slides2010/Martinsson.pdf).
```
Comment (wschin) [inline src/Microsoft.ML.PCA/PcaTrainer.cs]: My wish is that user can see the scoring rule in this document.

Comment (yaeldekel) [reply src/Microsoft.ML.PCA/PcaTrainer.cs]: I added it, but are you sure this is needed? isn't it too much math?

**Comment by shmoradims** [comment]
Duplicate of #3429. Closing for now.

**Pull request closed without merging**